RInAppMessaging/Classes/Extensions/Int32+IAM.swift

```diff
@@ -0,0 +1,14 @@
+import Foundation
+
+extension Int32 {
+    /// Exponential backoff
+    mutating func increaseBackOff() {
+        self = multipliedReportingOverflow(by: 2).partialValue
+    }
+
+    /// Exponential backoff with a random value
+    mutating func increaseRandomizedBackoff() {
+        increaseBackOff()
+        self += Int32.random(in: 1..<60)
```

Comment:
Those are milliseconds so it should be 1..<60000
Also could you put upper bound (60) in a separate var (static let even) to indicate what it is and also to clarify time unit.
Also I'm not sure if it isn't better to get upper bound as a parameter. That way we won't force this method to work only with MS

Reply:
Can put it in the Constants.swift file I think

Reply:
Actually it is 1000..<60000 (between 1 second and 60 seconds)

Reply:
Done.